tests/testthat/test-hypothesize.R

```diff
@@ -70,7 +70,7 @@ test_that(
   "hypothesize() allows partial matching of null arg for point", {
   hyp_p <- mtcars_df %>%
     specify(response = mpg) %>%
-    hypothesize(null = "p", mu = 0)
+    hypothesize(null = "po", mu = 0)
```

Comment:
This PR breaks partial matching of the `null` argument when supplied `null = "p"`. This is fine with me, but worth surfacing here (and maybe noting in NEWS?)

Reply:
So, previously, if `null = "p"`, what did that match to?